ui/tests/acceptance/cluster-test.js

```diff
@@ -31,15 +31,16 @@ const authAccessor = async function (path) {
   await consoleComponent.runCommands([`write auth/${path}/users/end-user password="${PASSWORD}"`]);
 };
 
-const setupUser = async function () {
-  const authMethodPath = `userpass-${new Date().getTime()}`;
+const setupUser = async function (uid) {
+  const authMethodPath = `userpass-${uid}`;
   await authAccessor(authMethodPath);
 };
 
 module('Acceptance | cluster', function (hooks) {
   setupApplicationTest(hooks);
 
   hooks.beforeEach(async function () {
+    this.timestamp = new Date().getTime();
```

Comment:
same here - can we use the `ui/lib/core/addon/utils/timestamp.js `util here or is this supposed to be a static date since it's a test?

Reply:
We could -- one caveat is that if the test stubs timestamp then we need to use this syntax to get a non-static date. Timestamp.now in this scenario would be the same as new date so here it would be `this.timestamp.now().getTime()` which doesn't feel any cleaner. 

Another thing we could do, since it's just supposed to be a unique value, is to use `uuidv4()` instead. I'm leaning toward this along with more specific naming within the test so that it's easier to track down within the vault logs which test is triggering the log. What do you think of that?

Reply:
I like the idea of using a `uuidv4()`.  It's also consistent with the pattern that's set up, making it easier to debug. :)